Makefile

```diff
@@ -369,19 +397,12 @@ $(BIN)/gotestsum: Makefile go.mod
 	@mkdir -p $(@D)
 	GOBIN=$(abspath $(@D)) $(GO) install gotest.tools/gotestsum@v1.9.0
 
-DLV_VERSION=v1.23.0
-
-$(BIN)/dlv: Makefile go.mod
+dlv:
+    # dlv is not intended for local use and is to be installed in the
+    # platform-specific docker image together with the main Pyroscope binary.
 	@mkdir -p $(@D)
-	GOBIN=$(abspath $(@D)) CGO_ENABLED=0 $(GO) install -ldflags "-s -w -extldflags '-static'" github.com/go-delve/delve/cmd/dlv@$(DLV_VERSION)
-
-$(BIN)/linux_amd64/dlv: Makefile go.mod
-	GOOS=linux GOARCH=amd64 CGO_ENABLED=0 GOPATH=$(CURDIR)/.tmp $(GO) install -ldflags "-s -w -extldflags '-static'" github.com/go-delve/delve/cmd/dlv@$(DLV_VERSION);
-	# Create a hardlink if you are on linux_amd64, so we are able to use the same dockerfile
-	if [[ "$(shell $(GO) env GOOS)" == "linux" && "$(shell $(GO) env GOARCH)" == "amd64" ]]; then \
-		mkdir -p "$(@D)"; \
-		ln -f $(BIN)/dlv "$@"; \
-	fi
```

Comment:
I'm not very sure if anyone relies on this, therefore removing it to avoid confusion with the cross-platform dlv we install in the image.

Reply:
I am not relying on it anymore, can really remember why I added it